### PR TITLE
ui: add clipboard to app

### PR DESCRIPTION
**Pull request by edouardparis**
ClipboardContext is added to ui::App
using https://github.com/alacritty/copypasta
which have clipboard support for
Windows, Mac OSX, GNU/Linux, and FreeBSD.

Support was tested with:

- Xorg    | success
- Wayland | failed

### CODE REVIEW

File: src/ui/app.rs [hunk 1]
```diff
@@ -91,6 +93,10 @@ impl Application for App {
             }
             Message::ChangeRole(role) => self.load_state(role, self.context.menu.to_owned()),
             Message::Menu(menu) => self.load_state(self.context.role, menu),
+            Message::Clipboard(text) => {
+                self.clipboard.set_contents(text).unwrap();
```
Comment (edouardparis) [inline src/ui/app.rs:97]: log the error

File: src/ui/view/vault.rs [hunk 2]
```diff
@@ -87,7 +89,20 @@ impl VaultModal {
...
+                                                                        ))
+                                                                        .push(button::clipboard(
+                                                                            &mut self.copy_button,
+                                                                            Message::Clipboard(
+                                                                                vlt.txid
+                                                                                    .to_string(),
+                                                                            ),
+                                                                        ))
+                                                                        .align_items(Align::Center),
+                                                                )
```
Comment (darosior) [inline src/ui/view/vault.rs:105]: Ok; so the intent is to be able to copy the deposit txid.

**Review by darosior — APPROVED** [review]
ACK ab0aa913eb223a629be66890f0f33d16bf6d4c42

File: src/ui/view/vault.rs (hunk 2 above)
Comment (edouardparis) [reply src/ui/view/vault.rs:105]: I use this as an example, but this button will be added where our customers will ask us for better user experience

**Pull request merged**